spec/features/workflow/schedule_without_selection_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.feature "Attempting to schedule without confirming a review status" do
+  scenario do
+    given_there_is_an_edition_in_draft
+    when_i_visit_the_summary_page
+    and_try_and_schedule_without_selecting_a_review_status
+    then_an_error_is_displayed
+  end
+
+  def given_there_is_an_edition_in_draft
+    @edition = create(:edition, scheduled_publishing_datetime: Time.current.tomorrow)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def and_try_and_schedule_without_selecting_a_review_status
+    click_on "Schedule"
+    stub_default_publishing_api_put_intent
```

Comment:
Hopefully we shouldn't need a stub here either :-).

Reply:
True true true